src/app/year-calendar/year-calendar-config.ts

```diff
@@ -0,0 +1,28 @@
+import {locale, Moment} from 'moment';
+import {ICalendar, ICalendarInternal} from '../common/models/calendar.model';
+import {ECalendarValue} from '../common/types/calendar-value-enum';
+
+export interface IConfig {
+  isYearDisabledCallback?: (date: Moment) => boolean;
+  allowMultiSelect?: boolean;
+  yearFormat?: string;
+  yearFormatter?: (month: Moment) => string;
+  format?: string;
+  isNavHeaderBtnClickable?: boolean;
+  yearBtnFormat?: string;
+  yearBtnFormatter?: (day: Moment) => string;
+  yearBtnCssClassCallback?: (day: Moment) => string;
+  multipleYearsNavigateBy?: number;
```

Comment:
what will this do in a year picker?

Reply:
Again, what are you refering to here? All the four lines, or any one in particular?

Reply:
`multipleYearsNavigateBy` & `showMultipleYearsNavigation`